src/style.css

```diff
@@ -324,3 +330,157 @@ input:checked + .hearth {
     background-position-x: calc(var(--heart-size) * (var(--frames) * -1 + 3));
   }
 }
+
+/* ****************popup************* */
+
+.popup {
+  position: fixed;
+  padding: 2rem;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+  min-height: 100vh;
+  backdrop-filter: blur(8px);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.popup-card {
+  position: relative;
+  background-color: rgb(255, 255, 255, 0.85);
+  border-radius: 7px;
+  border-top-right-radius: 20vh;
+  height: 85vh;
+  width: 100%;
+  color: #1b1b1b;
+  padding: 3rem 2rem;
+}
+
+#closeBtn {
+  position: absolute;
+  top: 1rem;
+  left: 1rem;
+  width: 2rem;
+}
+
+.meal-image {
+  position: absolute;
+  right: 0;
+  top: 0;
+  max-width: 50%;
+  height: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.popup-card h2 {
+  font-weight: 100;
+  color: #454545;
+}
+
+.food-name {
+  font-weight: 700;
+  font-size: 3rem;
+}
+
+.food-ingredients span {
+  background-color: #de7800;
+  border-radius: 10px;
+  color: #fff;
```

Comment:
It would be better and more re-useable if you made all the colors be from variables in `::root`